### ISSUE TITLE
[Bug]: Uncaught TypeError: The "path" ...

### ISSUE THREAD

**Issue by dajotim937** [ollm/OpenComic#235]
### Preflight Checklist

- [X] I have searched the [issue tracker](https://www.github.com/ollm/OpenComic/issues) for a bug report that matches the one I want to file, without success.

### OpenComic Version

1.2.0

### Operating System

Win10, 22H2, build 19045

### Steps to reproduce

To be honest, I caught that error almost week ago so I don't remember exactly what was cause of it.
What I remember: I read cbz files in folder (added to library via master folder). Then I renamed them (from "<name manga> - c0**" to "c0**") and after that when I opened folder in app, I got that error

### Expected Behavior

Not having error.

### Actual Behavior

```
Uncaught TypeError: The "path" argument must be of type string. Received undefined
    at validateString (node:internal/validators:162:11)
    at Object.normalize (node:path:309:5)
    at Readable.<anonymous> (C:\Users\***\AppData\Local\Programs\OpenComic\resources\app.asar\scripts\file-manager.js:1879:34)
    at Readable.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:559:12)
    at readableAddChunkPushObjectMode (node:internal/streams/readable:536:3)
    at Readable.push (node:internal/streams/readable:391:5)
    at Object.bodyData (C:\Users\***\AppData\Local\Programs\OpenComic\resources\app.asar\node_modules\node-7z\src\maybe.js:77:12)
    at C:\Users\***\AppData\Local\Programs\OpenComic\resources\app.asar\node_modules\node-7z\src\events.js:83:11
    at Socket.<anonymous> (C:\Users\***\AppData\Local\Programs\OpenComic\resources\app.asar\node_modules\node-7z\src\lifecycle.js:64:51)
```

<details>
<summary>Screenshots</summary>

![2024-06-04_03-14-26](https://github.com/ollm/OpenComic/assets/96221151/da51d5ae-8c0c-4ffd-84bb-843d8d9e47e7)
![2024-06-04_03-14-22](https://github.com/ollm/OpenComic/assets/96221151/dab90987-5ac5-41c7-a9d3-6a6c5aa3f75a)
![2024-06-04_03-16-33](https://github.com/ollm/OpenComic/assets/96221151/dd18489c-e281-4da7-824e-c9ae14a4d3b7)

</details

**Comment by dajotim937** [comment]
So yeah, got same error again.
1. Folder with bunch of cbz files with name "## - Ep.#"
2. Read 10 of them.
3. Mass rename to "Ep.#"
4. Open folder in OpenComic again and got the error.
5. Cleared temp and cash, instead of many errors got only one.

**Comment by ollm** [comment]
At the moment I have not been able to reproduce the error, from what i see, it seems that the error occurs when reading the cbz file using 7zip.

Can you try the following build? Has a fix related to 7zip when reading certain files 27c863b6a9abd434e8855216100e5f8087ed1e73

Build: https://mega.nz/file/zbx3maAK#UP67deOXFB39shZGfinJgTYjhuH2JRri_v-sVMoBUug

**Comment by ollm** [comment]
Closed as fixed and available in [v1.3.0](https://github.com/ollm/OpenComic/releases)
